src/com/lftechnology/java/training/sanish/game/FootballLeague.java

```diff
@@ -0,0 +1,28 @@
+package com.lftechnology.java.training.sanish.game;
+
+import java.util.ArrayList;
+import java.util.logging.Logger;
+
+/**
+ * Football League game result display program
+ *
+ * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+ */
+public class FootballLeague {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)